optimum/commands/export/neuronx.py

```diff
@@ -85,6 +85,11 @@ def parse_args_neuronx(parser: "ArgumentParser"):
         type=Path,
         help="Path indicating the directory where to store intermediary files generated by Neuronx compiler.",
     )
+    optional_group.add_argument(
+        "--disable-weights-neff-inline",
```

Comment:
```suggestion
        "--weights-neff-inline",
```

Reply:
According to the help string it seems that when set you want to actually inline weights no?

Reply:
I will change a bit the help to make it clear.
In the beginning, I wanted to separate weights / neff by default, then I hit some edge cases eg. some models arch are not well supported, dynamic batching not yet supported but will come in the future. So I prefer to inline weight by default for now.